components/skip-link/src/stories.tsx

```diff
@@ -16,7 +16,7 @@ class AutoFocussed extends Component {
   }
 
   componentDidMount() {
-    this.focusableRef.current.focus();
+    this.focusableRef.current?.focus();
```

Comment:
in storyshots, the previous version of this fails complaining that `current` is null - this fixes